samples/tab-external-auth/csharp/TabExternalAuth/Models/UserJsonMapper.cs

```diff
@@ -0,0 +1,32 @@
+﻿// <copyright file="UserJsonMapper.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+using Newtonsoft.Json;
+
+namespace TabExternalAuth.Models
+{
+    /// <summary>
+    /// Model to map response json to user object.
+    /// </summary>
+    public class UserJsonMapper<T>
+    {
+        /// <summary>
+        /// User's display name.
+        /// </summary>
+        [JsonProperty("names")]
+        public object Names { get; set; }
+
+        /// <summary>
+        /// User's profile photo.
+        /// </summary>
+        [JsonProperty("photos")]
+        public object Photos { get; set; }
+
+        /// <summary>
+        /// User's email address.
+        /// </summary>
+        [JsonProperty("emailAddresses")]
+        public object EmailAddresses { get; set; }
```

Comment:
Only one email address

Reply:
Its as per the json recieved from google api, so that name is required for mapping